hs_core/templates/search/search.html

```diff
@@ -0,0 +1,141 @@
+<form method="get" action=".">
+    <table>
+        <tbody>
+            {{ form.as_table }}
+            <tr>
+                <td>&nbsp;</td>
+                <td><input type="submit" value="Search"></td>
+            </tr>
+        </tbody>
+    </table>
+</form>
+
+{% if query %}
+    <!-- Begin faceting. -->
+
+        <!-- Display results... -->
+    <h2>Resource Results</h2>
+    {% for result in page.object_list %}
+        <div class="search_result">
+            <h3><a href="{{ result.object.get_absolute_url }}">{{ result.object.title }}</a></h3>
+
+            <p>{{ result.object.body|truncatewords:80 }}</p>
+        </div>
+    {% empty %}
+        <p>Sorry, no results found.</p>
```

Comment:
Is this search.html template used/included in hydroshare page main template to integrate search/discovery in hydroshare user interface? I don't see it is referred in any hydroshare page template although I could miss something.